CHANGELOG.md

```diff
@@ -3,9 +3,15 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
 - `TableauApi::Resources::Groups` added to support API calls for adding/deleting/updating groups.
-- `TableauApi::Resources::Workbook#permissions` now supports group permissions.
 - `TableauApi::Resources::Workbook#remove_permissions` added, including support for user and group permissions.
+- `TableauApi::Resources::Workbook#permissions` rewritten to retrieve instead of set permissions.
+- `TableauApi::Resources::Workbook#add_permissions` supports group permissions.
+- `TableauApi::Resources::Workbook#add_permissions` added, replaces `#permissions` for adding permissions.
+
+### Broken
```

Comment:
http://keepachangelog.com mentions `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, and `Security`, but not `Broken`. This seems to line up best with `Changed`, though I also see a case for `Removed`.